index.bs

```diff
@@ -12751,6 +12751,68 @@ type is used for representing function values that take no arguments and do not
 return any value.
 
 
+<h3 id="TextDirection" enum>TextDirection</h3>
+
+<pre class="idl">
+    enum TextDirection {
+      "auto",
+      "ltr",
+      "rtl"
+    };
+</pre>
+
+The {{TextDirection}} [=enumerable=] is used to represent text directionality as described in [[!BIDI]].
+
+
+<h3 id="localizable" dictionary>Localizable</h3>
+
+A <dfn export>localizable member</dfn> is a [=dictionary member=] that
+represents a bidirectional algorithm paragraph when displayed,
+as defined by the bidirectional algorithm’s rules P1, P2, and P3,
+including, for instance, supporting the paragraph-breaking behavior
+of <span class="char">U+000A LINE FEED (LF)</span> characters. [[!BIDI]]
+
+A user agent is expected to honor the Unicode semantics of [=localizable members=].
+
+The [=identifier=] of a [=localizable member=] is left to the defining specification (e.g., "title").
+
+Dictionaries that specify a [=localizable member=] must inherit from the {{Localizable}} dictionary.
+
+Specification authors must specify in prose which [=dictionary members|member(s)=]
+of the inheriting dictionary serve as [=localizable members=].
+
+Specification authors should limit the number of [=localizable members=] to one per dictionary,
+including in [=inherited dictionaries=] (if any).
```

Comment:
Maybe we should go a step further and define

```webidl
dictionary LocalizedValue : Localizable {
  DOMString value;
}

typedef (DOMString or LocalizedValue) LocalizableString;
```

and then encourage

```webidl
dictionary Thing {
  LocalizableString label;
  LocalizableString value;
}
```

? That seems better than each consumer having to define its own dictionary.